packages/angular_devkit/build_angular/src/tools/esbuild/application-code-bundle.ts

```diff
@@ -65,9 +65,8 @@ export function createBrowserCodeBundleOptions(
   }
 
   if (options.externalPackages) {
-    // Package files affected by a customized loader should not be implicitly marked as external
+    // Package files affected by a customized plugin should not be implicitly marked as external
     if (
-      options.loaderExtensions ||
```

Comment:
This is by design, as in some cases loaders need to be configured and applied to node packages.

Reply:
Then we stick to webpack I guess, as the esbuild 6 second watcher rebuilds is a regression.